docs/docs/apis/actions/introduction.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Actions
+---
+
+This page describes the options you have when writing ZITADEL actions scripts.
```

Comment:
This is the exact same content as docs/apis/actions.md ? 

I would suggest to de-duplicate the content: Never good for google search, the reader (like me) is confused about the deja-vu, and you need to maintain the content two times. 

We could remove this page imo. Alternatively make a sub-page and include it in both documents that it's always updated where copied (see examples in deploy guides).

Reply:
Yes that's my fault. I splitted the action.md into these files but forgot to remove the actions.md